examples/deprecation/deprecation.bal

```diff
@@ -0,0 +1,49 @@
+// The constant 'CONST1' is deprecated using '@deprecated' annotation and markdown documentation '# # Deprecated'
+# Define a constant
+# # Deprecated
+# This constant is deprecated
+@deprecated
+const string CONST1 = "CONST1";
+const string CONST2 = "CONST2";
```

Comment:
Instead of using names like `CONST1`, `CONST2`, `Foo`, `DummyObject`, shall we try and come up with a more practical and complete example?

Reply:
Refer to the example given at https://github.com/ballerina-platform/ballerina-spec/blob/master/langext/deprecation/spec.md

Reply:
Done